examples/pylint/C0301_line_too_long.py

```diff
@@ -0,0 +1 @@
+TEMP = "I am in the file C0301_line_too_long.py. I raise the error for a very long line."
```

Comment:
instead of,
I raise the error for a very long line.

maybe say like,
This example raises an error for exceeding the acceptable line length.

i'm not sure if we should use "I"